### PR TITLE
feat: logowanie do serwisu

### PR DESCRIPTION
**Pull request by N33Qu**
Please rewiew MS

### CODE REVIEW

File: src/Login.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    public String getUsername() {
+        return username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public String toString() {
+        return "Username: " + this.username + " Password: " + this.password + "\n" +
```
Comment (Valentine0604) [inline src/Login.java]: what is this, its not a xmas time yet! please change it to sth related to halloween and cats, 
thank you
Valentine

File: src/Login.java [hunk 2]
```diff
@@ -17,13 +17,12 @@ public String getPassword() {
     }
 
     public String toString() {
```
Comment (Valentine0604) [inline src/Login.java:19]: ITS NEITHER CAT NOR PUMPKIN! but leave it okok

File: src/Login.java (hunk 1 above)
Comment (N33Qu) [reply src/Login.java]: done.

File: src/Login.java (hunk 2 above)
Comment (N33Qu) [reply src/Login.java:19]: ok

**Review by Valentine0604 — APPROVED** [review]

**Pull request merged**